src/main/resources/carpet.mixins.json

```diff
@@ -4,212 +4,177 @@
+    "ServerPlayerInteractionManager_cactusMixin",
+    "ServerPlayerInteractionManager_scarpetEventsMixin",
+    "ServerPlayNetworkHandler_antiCheatDisabledMixin",
+    "ServerPlayNetworkHandler_coreMixin",
+    "ServerPlayNetworkHandler_interactionUpdatesMixin",
+    "ServerPlayNetworkHandler_scarpetEventsMixin",
+    "ServerPlayNetworkHandler_tickMixin",
+    "ServerPlayNetworkHandlerMixin",
+    "ServerWorld_fakePlayersMixin",
+    "ServerWorld_onePlayerSleepingMixin",
+    "ServerWorld_scarpetMixin",
+    "ServerWorld_spawnChunksMixin",
+    "ServerWorld_tickMixin",
+    "SetBlockCommandMixin",
+    "SkeletonHorseTrapTriggerGoal_extremeMixin",
+    "SkyLightStorage_scarpetChunkCreationMixin",
+    "SpawnHelperInnerMixin",
+    "SpawnHelperMixin",
     "StatePredicate_scarpetMixin",
+    "StatType_scarpetMixin",
+    "Structure_fillUpdatesMixin",
+    "StructureBlockBlockEntityMixin",
+    "StructureFeatureMixin",
+    "StructurePiece_scarpetPlopMixin",
+    "SummonCommandMixin",
     "TagPredicate_scarpetMixin",
-    "HorseBaseEntity_scarpetMixin",
-    "PillagerEntity_scarpetMixin",
-
-    "PistonBlock_movableTEMixin",
-    "PistonBlockEntity_movableTEMixin",
+    "ThreadedAnvilChunkStorage_scarpetChunkCreationMixin",
+    "ThreadedAnvilChunkStorage_tickMixin",
+    "ThrownEnderpearlEntity_fakePlayersMixin",
+    "ThrownEntityMixin",
+    "TntBlockMixin",
+    "TntEntityMixin",
+    "TradeOutputSlot_scarpetEventMixin",
+    "UpdateStructureBlockC2SPacketMixin",
+    "VillagerEntity_aiMixin",
+    "WeightedPickerEntryMixin",
+    "World_fillUpdatesMixin",
     "World_movableTEMixin",
-    "WorldChunk_movableTEMixin",
-    "PistonHandler_customStickyMixin",
-    "PlayerEntity_creativeNoClipMixin",
-    "PistonBlockEntity_playerHandlingMixin",
-    "BlockItem_creativeNoClipMixin",
-    "LivingEntity_creativeFlyMixin"
-
+    "World_scarpetPlopMixin",
+    "World_tickMixin",
+    "WorldChunk_fillUpdatesMixin",
+    "WorldChunk_movableTEMixin"
   ],
   "client": [
-    "RenderTickCounter_tickSpeedMixin",
-    "MinecraftClientMixin",
-    "MinecraftClient_pausedShakeMixin",
-    "WorldRenderer_pausedShakeMixin",
-    "InGameHudMixin",
-    "PlayerListHudMixin",
-    "PistonBlockEntityRenderer_movableTEMixin",
     "ClientPlayerEntity_clientCommandMixin",
-    "WorldRenderer_fogOffMixin",
-    "WorldRenderer_creativeNoClipMixin",
     "ClientPlayNetworkHandlerMixin",
     "DebugRenderer_scarpetRenderMixin",
-    "WorldRenderer_scarpetRenderMixin",
-
+    "InGameHudMixin",
+    "MinecraftClient_pausedShakeMixin",
+    "MinecraftClientMixin",
+    "PistonBlockEntityRenderer_movableTEMixin",
+    "PlayerListHudMixin",
+    "RenderTickCounter_tickSpeedMixin",
     "SoundEngineInt_cleanLogsMixin",
-    "SoundSystem_cleanLogsMixin"
+    "SoundSystem_cleanLogsMixin",
+    "WorldRenderer_creativeNoClipMixin",
+    "WorldRenderer_fogOffMixin",
+    "WorldRenderer_pausedShakeMixin",
+    "WorldRenderer_scarpetRenderMixin"
```

Comment:
Looks like you sorted the mixins file

Reply:
I had that too with the Minecraft Dev idea plugin when generating accessor and invoker mixins

Reply:
Didn't notice that. I will change it back in the next commit.